packages/devtools_app/lib/src/app_size/file_import_container.dart

```diff
@@ -138,18 +140,35 @@ class _FileImportContainerState extends State<FileImportContainer> {
     );
```

Comment:
not your fault, but can you make the Text widget in _buildImportedFileDisplay ellipsize instead of wrap to a second line?